thirdeye-server/src/main/java/ai/startree/thirdeye/core/AppAnalyticsService.java

```diff
@@ -79,4 +100,32 @@ private MonitoredMetricWrapper wrapMonitoredMetric(final AlertMetadataDTO metada
         .setDataset(metadata.getDataset().getName())
         .setDatasource(metadata.getDatasource().getName());
   }
+
+  public ConfusionMatrix computeConfusionMatrixForAnomalies() {
+    int truePositive = 0;
+    int falsePositive = 0;
+    int unclassified;
+    List<MergedAnomalyResultDTO> anomalies = anomalyManager.findAll();
```

Comment:
I think child anomalies should not be counted. I think there is no opportunity for the user to give feedback on them.

Reply:
understand they are filtered after but filtering at the db level will have better performance

Reply:
Make sense. Done.